Fix typo in czech translation

Trsak opened the pull request.

Trsak committed:
Fix typo in czech translation

tabarra reviewed (commented):
is `dr tento` correct?

Trsak commented:
> is `dr tento` correct?

No, it is not. I changed first sentence to have better structure and fixed `dr tento` TYPO.
There was an TYPO in second sentence aswell: `odpoje` is wrong, should be `odpojte`.
